src/controllers/midi/midicontroller.cpp

```diff
@@ -279,7 +279,7 @@ void MidiController::processInputMapping(const MidiInputMapping& mapping,
                 status,
                 mapping.control.group,
         };
-        if (!pEngine->executeFunction(function, args)) {
+        if (!pEngine->executeFunction(std::move(function), args)) {
```

Comment:
This is conflicting with my change in https://github.com/mixxxdj/mixxx/pull/11701 making it a pointer to fix clang-tidy Qt6 warnings.